tests/testing/test_results.py

```diff
@@ -151,6 +151,39 @@ def test_simple_atol(self, actual, atol, expected_issues):
         expected = xarray.DataArray([1, 2, 3])
         assert _compare_xarray_dataarray(actual=actual, expected=expected, rtol=0, atol=atol) == expected_issues
 
+    @pytest.mark.parametrize(
+        ["actual", "atol", "pixel_tolerance", "expected_issues"],
+        [
+            (xarray.DataArray([1, 2, 3.001, 4, 5]), 1e-4, 21.0, []),
```

Comment:
make it more pronounced (here and in the other cases) that we are talking about significant differences:
```suggestion
            (xarray.DataArray([1, 2, 300, 4, 5]), 1e-4, 21.0, []),
```